schema.sql

```diff
@@ -1,51 +1,57 @@
-CREATE TABLE animals (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    date_of_birth DATE,
-    escape_attempts INT,
-    neutered BOOLEAN,
-    weight_kg DECIMAL(5, 2)
+CREATE TABLE owners(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  full_name VARCHAR(20),
+  age INT,
+  PRIMARY KEY(id)
 );
 
-ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
-
-CREATE TABLE owners (
-    id SERIAL PRIMARY KEY,
-    full_name VARCHAR(255),
-    age INTEGER
+CREATE TABLE species(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  PRIMARY KEY(id)
 );
 
-CREATE TABLE species (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255)
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  date_of_birth DATE,
+  escape_attempts INT,
+  neutered BOOLEAN,
+  weight_kg DECIMAL,
+  species_id INT REFERENCES species(id),
+  owner_id INT REFERENCES owners(id),
+  PRIMARY KEY(id)
 );
 
-ALTER TABLE animals
-    DROP COLUMN species,
-    ADD COLUMN species_id INTEGER,
-    ADD COLUMN owner_id INTEGER;
-
-ALTER TABLE animals
-    ADD CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id),
-    ADD CONSTRAINT fk_owner FOREIGN KEY (owner_id) REFERENCES owners(id);
-
-CREATE TABLE vets (
-    id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    age INTEGER,
-    date_of_graduation DATE
+CREATE TABLE vets(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(20),
+  age INT,
+  date_of_graduation DATE,
+  PRIMARY KEY(id)
 );
 
-CREATE TABLE specializations (
-    vet_id INTEGER REFERENCES vets(id),
-    species_id INTEGER REFERENCES species(id),
-    PRIMARY KEY (vet_id, species_id)
+CREATE TABLE specializations(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  species_id INT REFERENCES species(id),
+  vet_id INT REFERENCES vets(id),
+  PRIMARY KEY(id)
 );
 
-CREATE TABLE visits (
-    id SERIAL PRIMARY KEY,
-    animal_id INTEGER REFERENCES animals(id),
-    vet_id INTEGER REFERENCES vets(id),
-    date_of_visit DATE
+CREATE TABLE visits(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  animal_id INT REFERENCES animals(id),
+  vet_id INT REFERENCES vets(id),
+  date_of_visit DATE,
+  PRIMARY KEY(id)
 );
+
+<<<<<<< HEAD
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX index_animal_id ON visits(animal_id);
+
+CREATE INDEX index_vet_id ON visits(vet_id);
+
+CREATE INDEX index_email ON owners(email);
```

Comment:
- [ ] Great work so far dear 👍 but I guess you missed this thing here that the execution time in the before screenshot should be more than 1000ms but i can see that it is lower than 1000ms in the before screenshot 😨 Kindly populate your database to increase your execution time 😊 to follow the project requirement as suggested by the previous reviewer 👀